.github/workflows/ci-conda-workflow.yml

```
@@ -104,23 +105,69 @@ jobs:
         XSPECVER: ${{ matrix.xspec-version }}
       run: |
         source .github/scripts/setup_conda.sh
-        if [ -n "${XSPECVER}" ] ; then
-            source .github/scripts/setup_xspec_ds9.sh
-        fi
 
-    - name: Build Sherpa
+    - name: Conda Setup (Xspec and DS9)
+      shell: bash
+      if: matrix.xspec-version != ''
+      env:
+        XSPECVER: ${{ matrix.xspec-version }}
+      run: |
+        source ${miniconda_loc}/etc/profile.d/conda.sh
+        conda activate build
+        source .github/scripts/setup_xspec_ds9.sh
+
+    - name: Build Sherpa (develop)
+      shell: bash
+      if: matrix.install-type == 'develop'
+      env:
+        PYTHON_LDFLAGS: " "
+      run: |
+        source ${miniconda_loc}/etc/profile.d/conda.sh
+        conda activate build
+        pip install -e .[test] --verbose
+
+    - name: Build Sherpa (install)
+      shell: bash
+      if: matrix.install-type == 'install'
       env:
         PYTHON_LDFLAGS: " "
       run: |
         source ${miniconda_loc}/etc/profile.d/conda.sh
         conda activate build
-        python setup.py ${{ matrix.install-type }}
+        pip install . --verbose
+
+    - name: install packaged test data
+      shell: bash
+      if: matrix.test-data == 'package'
+      run: |
+        source ${miniconda_loc}/etc/profile.d/conda.sh
+        conda activate build
+        pip install ./sherpa-test-data
+
+    - name: remove local test data
+      shell: bash
+      if: matrix.test-data == 'package' || matrix.test-data == 'none'
+      run: |
+        git submodule deinit -f .
+
+    - name: freeze
+      shell: bash
+      env:
+        TEST: ${{ matrix.test-data }}
+      run: |
+        source ${miniconda_loc}/etc/profile.d/conda.sh
+        conda activate build
+        pip install pytest-xvfb pytest-cov codecov
```

When you rebase you'll want to drop the codecov install on the merge.
```suggestion
        pip install pytest-xvfb pytest-cov
```